google_drive/google_docs.py

```diff
@@ -1,33 +1,42 @@
+"""
+Google Document XBlock implementation
+"""
 # -*- coding: utf-8 -*-
 #
 
 # Imports ###########################################################
-
-import pkg_resources
-#import logging
+import logging
 import textwrap
-#import json
-#import webob
-#from lxml import etree
-#from xml.etree import ElementTree as ET
+import requests
 
 from xblock.core import XBlock
 from xblock.fields import Scope, String
 from xblock.fragment import Fragment
 
-# from StringIO import StringIO
-
-from .utils import render_template, AttrDict, load_resource
-
+from xblockutils.publish_event import PublishEventMixin
+from xblockutils.resources import ResourceLoader
 
-# Globals ###########################################################
+LOG = logging.getLogger(__name__)
+RESOURCE_LOADER = ResourceLoader(__name__)
 
-#log = logging.getLogger(__name__)
+# Constants ###########################################################
+DEFAULT_EMBED_CODE = textwrap.dedent("""
+            <iframe
+                src="https://docs.google.com/presentation/d/1x2ZuzqHsMoh1epK8VsGAlanSo7r9z55ualwQlj-ofBQ/embed?start=true&loop=true&delayms=10000"
+                frameborder="0"
+                width="960"
+                height="569"
+                allowfullscreen="true"
+                mozallowfullscreen="true"
+                webkitallowfullscreen="true">
+            </iframe>
+        """)
```

Comment:
I too was surprised, but this was considered the best MVP that we could deliver, and hope to follow up with a better experience later